velox/substrait/SubstraitToVeloxPlan.h

```diff
@@ -107,6 +112,34 @@ class SubstraitVeloxPlanConverter {
   /// name>:<arg_type0>_<arg_type1>_..._<arg_typeN>
   const std::string& findFunction(uint64_t id) const;
 
+  /// Substrait emit feature integration. Here a given substrait::Rel is passed
+  /// and check if emit is defined for this relation. If so, the emit fields
+  /// are extracted from the relation. To simulate an emit, a projection is used
+  /// to pick the fields which is selected to be emitted.
+  template <typename RelMessage>
+  core::PlanNodePtr ProcessEmit(
```

Comment:
naming: ProcessEmit -> processEmit

Does this need to be a template?

Reply:
Same thought here: https://github.com/facebookincubator/velox/pull/3543#discussion_r1054056863 
WDYT?

Reply:
Similarly, to the other method, I wonder if this method can take rel.common() instead of rel.

Reply:
That's fair, although we will have to check if it has common in each relation. Is that okay?
I guess the motiviation is avoid templates as much as possible (I assume readability and maintainability)?

Reply:
It would be best to limit usage of templates. Template methods have to be defined in the .h file which increases build times and increases the size of the binary.

Reply:
I understand. I will make the necessary changes and remove the templates. Thanks for the heads up, would be very useful in writing other PRs.

Reply:
Templates removed.